manifests/node-maintenance-operator/template/node-maintenance-operator.clusterserviceversion.yaml

```diff
@@ -32,10 +32,16 @@ spec:
       kind: NodeMaintenance
       name: nodemaintenances.nodemaintenance.kubevirt.io
       version: v1beta1
-  description: Node maintenance operator
+  description: |
+    Node Maintenance Operator
+
+    This operator will keep nodes cordoned and drained while a matching node maintenance CR exists.
+    This is useful when investigating problems with a machine, or performing an operation on the underlying failure that might result in node failure.
```

Comment:
same as above

Reply:
Fixed at this place, the others are generated based on this